rust/src/adapter/honeycomb.rs

```diff
@@ -1,16 +1,12 @@
+use prost::Message;
 use std::time::Duration;
 
 use crate::{Event, TraceEvent};
 use anyhow::Result;
 
-use super::{
-    otel_formatter::{OtelFormatter, ResourceSpan, Span},
-    Adapter, AdapterHandle,
-};
-
 pub use super::{
-    otel_formatter::{Attribute, Value},
-    AdapterMetadata,
+    otel_formatter::{opentelemetry, Attribute, OtelFormatter, Value},
```

Comment:
Same suggestion as the Datadog public interface above; let's limit what we make public to only what we know the end-user needs.
This one probably should export the `Attribute` and `Value` in addition to the `Adapter` and related types, but not the formatter things.